### PR TITLE
Added an example on mounting the rpi.raw image

### PR DESCRIPTION
**Pull request by e-minguez**
Also moved the "Writing to a USB stick" section below the injection one.

### CODE REVIEW

**Comment by kkaempf** [comment]
🤩 - need to test this script, thanks !

File: docs/quickstart-cli.md [hunk 1]
```diff
@@ -193,6 +187,25 @@ If you've mounted the USB stick with a file manager, this command should work to
...
+
+```shell showLineNumbers
+IMAGE=rpi.raw
+DEST=$(mktemp -d)
+
+SECTORSIZE=$(sfdisk -J ${IMAGE} | jq '.partitiontable.sectorsize')
+DATAPARTITIONSTART=$(sfdisk -J ${IMAGE} | jq '.partitiontable.partitions[1].start')
+sudo mount -o rw,loop,offset=$((${SECTORSIZE}*${DATAPARTITIONSTART})) ${IMAGE} ${DEST}
+sudo cp initial-registration.yaml ${DEST}/livecd-cloud-config.yaml
+sudo umount ${DEST}
```
Comment (kkaempf) [inline docs/quickstart-cli.md:200]:
```suggestion
sudo umount ${DEST}; rmdir ${DEST}
```

**Review by kkaempf — APPROVED** [review]
script works nicely 👍🏻

File: docs/quickstart-cli.md (hunk 1 above)
Comment (e-minguez) [reply docs/quickstart-cli.md:200]: Added as a new line, thanks for the suggestion!

**Pull request merged**